de.dlr.sc.virsat.model.extension.mechanical.test/src/de/dlr/sc/virsat/model/extension/mechanical/catia/CatiaImporterTest.java

```diff
@@ -67,28 +81,203 @@
 	// Visualisation elements
 	private Visualisation reactionWheelVisDefinition;
 
-	// Create JSON object
+	private static final int TEST_POS_X_PRODUCT = 1;
+	private static final int TEST_POS_Y_PRODUCT = 2;
+	private static final int TEST_POS_Z_PRODUCT = 3;
+
+	private static final int TEST_ROT_X_PRODUCT = 4;
+	private static final int TEST_ROT_Y_PRODUCT = 5;
+	private static final int TEST_ROT_Z_PRODUCT = 6;
+
+	private static final String TEST_SHAPE_PRODUCT = Visualisation.SHAPE_BOX_NAME;
+
+	private static final int TEST_SIZE_X_PART = 7;
+	private static final int TEST_SIZE_Y_PART = 8;
+	private static final int TEST_SIZE_Z_PART = 9;
+	private static final int TEST_RADIUS_PART = 10;
+
+	private static final long TEST_COLOR_PART = 30;
+	private static final String TEST_SHAPE_PART = Visualisation.SHAPE_BOX_NAME;
+
+	private static final String STL_TEST_FILENAME = "SomeGeometry.stl";
 
 	@Before
-	public void setUp() {
+	public void setUp() throws CoreException {
+		super.setUp();
 		conceptPS = loadConceptFromPlugin("de.dlr.sc.virsat.model.extension.ps");
 		conceptVis = loadConceptFromPlugin("de.dlr.sc.virsat.model.extension.visualisation");
 
+		addEditingDomainAndRepository();
+		editingDomain.getVirSatCommandStack().execute(new RecordingCommand(editingDomain) {
+
+			@Override
+			protected void doExecute() {
+				repository.getActiveConcepts().add(conceptPS);
+				repository.getActiveConcepts().add(conceptVis);
+			}
+		});
 		createTestTreeScenario();
 	}
 
+	@After
+	public void tearDown() throws CoreException {
+		super.tearDown();
+	}
+
+	@Test
+	public void testTransform() {
+
+		JsonObject rootObject = createMappedJsonObjectWithProductAndConfiguration();
+
+		// Do the import
+		CatiaImporter importer = new CatiaImporter();
+		Map<String, StructuralElementInstance> mapping = importer.mapJsonUuidToSEI(rootObject, configurationTree);
+		Command importCommand = importer.transform(editingDomain, rootObject, mapping);
+		editingDomain.getVirSatCommandStack().execute(importCommand);
+
+		assertTrue("Check if product values are imported", elementConfigurationReactionWheel1

```

Comment:
let's maybe create a variable for `elementConfigurationReactionWheel1.getFirst(Visualisation.class)` ?